GenumKit/Parsers/FontsFileParser.swift

```diff
@@ -69,9 +71,9 @@ public final class FontsFileParser {
             continue
           }
           guard UTTypeConformsTo(uti, "public.font") else { continue }
-          guard let font = CTFont.parseFontInfo(file) else { continue }
+          guard let fonts = CTFont.parseFontInfo(file) else { continue }
```

Comment:
This way, this line simply becomes `let fonts = CTFont.parseFontInfo(file)` and this simplifies the possible states: instead of having to deal with two empty cases (`nil` and `[]`) representing similar "no font found in this file" failure, we would now only have one case to consider (the empty array `[]` case) and using `[].forEach { … }` won't generate any entry in the output anyway.

_… and here goes your dummy commit to force Travis-CI to re-trigger a build_ 😆